Examples/calculator.lisp

```diff
@@ -138,3 +149,6 @@
              (list seven eight divide)
              (list nine zero result))))))
   (:top-level (calculator-frame-top-level . nil)))
+
+#+nil
+(calculator)
```

Comment:
clim examples are meant to be started from `clim-demo:demodemo`, also I don't see point in adding commented out function call.

if there is some reason to have it, please use `#+(or)`, not `#+nil`